AttackItem.h

```diff
@@ -1 +1,11 @@
-#include "Item.h"
\ No newline at end of file
+
+#pragma once
+
+#include "Item.h"
+#include "Character.h"
```

Comment:
Character isn't used in this header file, so you can forward declare it.
include in the cpp file.
- [ ] revise